docs/whats-new/dotnet-docs-2021-10-01.md

```diff
@@ -0,0 +1,116 @@
+- [Tutorial: Create a .NET console application using Visual Studio Code](../core/tutorials/with-visual-studio-code.md) - Update VS Code tutorial series for .NET 6
+- [How to use a JSON document, Utf8JsonReader, and Utf8JsonWriter in System.Text.Json](../standard/serialization/system-text-json-use-dom-utf8jsonreader-utf8jsonwriter.md) - Address System.Text.Json feedback from #24963
+- [New C# templates generate top-level statements](../core/tutorials/top-level-templates.md) - Altered the instructions for classic template
+- [Troubleshoot .NET tool usage issues](../core/tools/troubleshoot-usage-issues.md) - Add global tool troubleshooting info for 401 errors
+- [MSBuild reference for .NET SDK projects](../core/project-sdk/msbuild-props.md) - .NET 6 code analysis config options
+
+## Architecture guides
+
+### Updated articles
+
+- [What is Cloud Native?](../architecture/cloud-native/definition.md) - Updates for Architecting Cloud-Native .NET Apps for Azure
+
+## C# language
+
+### New articles
+
+- [Tutorial: Write a custom string interpolation handler](../csharp/whats-new/tutorials/interpolated-string-handler.md)
+
+### Updated articles
+
+- [Compiler Error CS0120](../csharp/language-reference/compiler-messages/cs0120.md) - Add `how to fix` section in examples of cs0120.md
+
+## F# language
+
+### New articles
+
+- [What's new in F# 6](../fsharp/whats-new/fsharp-6.md)
+
+## ML.NET
+
+### New articles
+
+- [Train a recommendation model using Model Builder](../machine-learning/tutorials/movie-recommendation-model-builder.md)
+
+## Community contributors
+
+The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).
+
+- [GitHubPang](https://github.com/GitHubPang) (11)
+- [forensicmike](https://github.com/forensicmike) - forensicmike1 (4)
+- [nojaf](https://github.com/nojaf) - Florian Verdonck (3)
+- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (3)
+- [poke](https://github.com/poke) - Patrick Westerhoff (3)
+- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (3)
+- [bschaeublin](https://github.com/bschaeublin) - Benjamin Schäublin (2)
+- [GKotfis](https://github.com/GKotfis) - Grzegorz Kotfis (2)
+- [KevRitchie](https://github.com/KevRitchie) - Kev Ritchie (2)
+- [PiotrJustyna](https://github.com/PiotrJustyna) - Piotr Justyna (2)
+- [wenz](https://github.com/wenz) - Christian Wenz (2)
+- [ankitwww](https://github.com/ankitwww) - Ankit Kashyap (1)
+- [doctordns](https://github.com/doctordns) - Thomas Lee (1)
+- [dulanov](https://github.com/dulanov) - Daulet Dulanov (1)
+- [EtienneRQT](https://github.com/EtienneRQT) (1)
+- [github-actions](https://github.com/github-actions) -  (1)
+- [jjanuszkiewicz](https://github.com/jjanuszkiewicz) - Jakub Januszkiewicz (1)
+- [jwood803](https://github.com/jwood803) - Jon Wood (1)
+- [kcratss](https://github.com/kcratss) (1)
```

Comment:
We're missing a name here?

Reply:
Some GitHub users don't have their name as part of the public profile. We respect that, and only print their user ID.